build-logic/convention/src/main/kotlin/com/revenuecat/purchases/android/buildlogic/convention/ConfigureApisFlavors.kt

```diff
@@ -0,0 +1,21 @@
+package com.revenuecat.purchases.android.buildlogic.convention
+
+import com.android.build.api.dsl.LibraryExtension
+import org.gradle.api.Project
+import org.gradle.kotlin.dsl.configure
+
+/**
+ * Configures the standard "apis" flavor dimension used across all library modules
+ */
+internal fun Project.configureApisFlavors() {
+    extensions.configure<LibraryExtension> {
+        flavorDimensions += "apis"
```

Comment:
We will need to add another one like this for the `billingclient` dimension, that may have bc8 (default) or bc7 flavors. This should be configured for the `purchases`, `purchases-ui` and `puchases-store-amazon` dependencies. No need for the `debugview` dependency. Sorry for the conflicts 🙏

Reply:
Thank you for the detailed histories! 🙏  I just resolved the conflicts and configured the `billingclient` dimension on [b011871](https://github.com/RevenueCat/purchases-android/pull/2827/commits/b011871d6d860cd79f2879f213f82be3fb1c2629).